features/command/src/main/java/org/apache/karaf/features/command/ListFeaturesCommand.java

```diff
@@ -49,12 +50,16 @@ public class ListFeaturesCommand extends FeaturesCommandSupport {
     @Option(name = "-o", aliases = {"--ordered"}, description = "Display a list using alphabetical order ", required = false, multiValued = false)
     boolean ordered;
 
+    @Option(name = "--repository", description = "Only list features from that repository", required = false, multiValued = false)
```

Comment:
The ticket suggested to use `name = "-r"`  but since then it was used for another option (--required). So I used full name here.